apps/confidential/confidential.go

```diff
@@ -224,6 +227,13 @@ func WithHTTPClient(httpClient ops.HTTPClient) Option {
 	}
 }
 
+// SendX5C specifies if x5c claim(public key of the certificate) should be sent to STS to enable Subject Name Issuer Authentication
```

Comment:
Same period thing.

Reply:
As a follow up, I am certainly not an expert on all the auth methods.  But it seems to me that various users tend to see they need to do X, so they look to enable X.  x5c doesn't mean much to the user.

Keegan and yourself would know better, but I might call this:

EnableSNIAuth()

Reply:
I think I agree with you. EnableSNIAuth really makes more sense. Right now this is how things are done in .NET and Java. @jmprieur Is sendX5C chosen for a specific reason? Can we use EnableSNIAuuth() instead here ?

Reply:
This is the result of compromises in API reviews, @abhidnya13.
It's not so much that we enable SNI Authentication: we enable the same kind of confidential client flows, but we send the public key of a subject/name issuer cert to ease the cert rotation.